### ISSUE TITLE
Improving performamce of scatter plot

### ISSUE THREAD

**Issue by ghansham** [mcidasv/mcidasv#128]
https://github.com/mcidasv/mcidasv/blob/59f7a27fb793369f6bec8e3329bdf3d1f42627ba/edu/wisc/ssec/mcidasv/control/ScatterDisplay.java#L802

If we make rgb displayable to use ImageRenderer3D (fast renderer) which is quite possible here, the performance of scatter plot can be increased manifold. The maskColorPalette needs to be accordingly modified to reflect RGBA with alpha set to 1.0.

**Comment by rmcarp** [comment]
Hi Ghansham,

Thanks for the suggestion.  I wrote up evaluating this as McIDAS-V 
inquiry 2698 <http://mcidas.ssec.wisc.edu/inquiry-v/?inquiry=2698>.  One 
of our programmers took a look at this yesterday and doesn't believe 
that it is a straightforward change.  If you have any additional 
suggestions on how to go about implementing this, please let us know and 
I'll add it to the inquiry.

Thanks again,
Bob Carp

**Comment by ghansham** [comment]
Yeah it is straightforward. :-) I can send you the modified version of the
java file. If you go through mcv forums, I have shared the modified version
as a plugin long back and kris sir has confirmed that its faster than the
original. Our dev environment is internetless so its difficult to go via
github route. Hehehe..

**Comment by ghansham** [comment]
See this post on mcidas forum:
http://mcidas.ssec.wisc.edu/forums/viewtopic.php?f=26&t=1321&hilit=+scatter&sid=7b179ee59752a714af328089c5263188

**Comment by ghansham** [comment]
Sorry its not kris sir, but hproe sir.

**Comment by rmcarp** [comment]
Thanks, Ghansham.  I picked up the ScatterPlugin2.jar plugin from your 
10/02/2013 post and put it in our inquiry data repository for our 
programmers to take a look at.  Just to verify, there have been no other 
changes to this plugin since 2013?

Thanks again,
Bob

**Comment by ghansham** [comment]
I will fetch latest java file from github and send the modified code.

**Comment by rmcarp** [comment]
Hi Ghansham,

I apologize if you sent this and I missed it, but could you please send 
the modified code?  I want to make sure we have the most up-to-date code 
when our programmers get to looking at this one.

Thanks!
Bob

**Comment by ghansham** [comment]
Of course..  i will do it today itself
